websites/A/AnimeOnsen/presence.ts

```diff
@@ -15,7 +15,7 @@ const presence = new Presence({
 		clientId: "826806766033174568"
 	}),
 	[, page] = window.location.pathname.split("/"),
-	$ = (selectors: string) => document.querySelectorAll(selectors),
+	$ = (selectors: string) => document.querySelector(selectors),
```

Comment:
Use a proper variable name

Reply:
It's proper(?) Shorthand function for jQuery-like querySelector (accepted in first PR)